src/main/drivers/pwm_output.c

```diff
@@ -480,10 +480,10 @@ void pwmToggleBeeper(void)
         pwmWriteBeeper(!beeperPwm.enabled);
 }
 
-void beeperPwmInit(IO_t io, uint16_t frequency)
+void beeperPwmInit(const beeperDevConfig_t *beeperConfig)
 {
-        const ioTag_t tag=IO_TAG(BEEPER);
-        beeperPwm.io = io;
+        const ioTag_t tag = beeperConfig->ioTag;
+        beeperPwm.io = IOGetByTag(tag);
```

Comment:
This dereferencing of `beeperDevConfig_t` should be done in `sound_beeper.c` to avoid `pwm_output` being dependent on beeper.